charts/kong-collectorapi/Chart.yaml

```diff
@@ -7,4 +7,4 @@ maintainers:
 - name: jackkav
   email: jack.kavanagh@konghq.com
 version: 0.1.8
```

Comment:
I think if you scroll down to the bottom of the cI it'll say that you need to bump the version here for each PR. There isn't a formal release process for this repo, until we plug in the kong chart registry. So just up the version here and edit the changelog to reflect a release. I'll update the docs to make that clearer.